### DISCUSSION TITLE
¿Un Ejemplo de código para conectar con la libreria para enviar una factura?

### DISCUSSION THREAD

**Discussion by zaqueoae** [Barnetik/tbai-php-lib#36] [General]
Muy buenas,
Llevo un buen rato mirando como funciona pero no soy capaz.

En la carpeta test está el bootstrap.php y supuestamente cargando el autoload ya debería poder llamar a funciones. ¿Cierto? 

¿Podríais colgar un pedazo de código en el que se pueda conectar con la librería y quizás un ejemplo llamando a una función?

Mil gracias.

**Comment by zaqueoae** [comment]
Esto es lo que he escrito para testear, pero no veo las clases necesarias:

```
<?php

require_once 'vendor/autoload.php';

//Esto lo escribo para saber si autload está bien cargado:
$included_files = get_included_files();
foreach ($included_files as $file) {
    echo $file . PHP_EOL;
}

//Esto lo escribo para ver las clases disponibles
$declared_classes = get_declared_classes();

foreach ($declared_classes as $class) {
    echo $class . PHP_EOL;
}
```

**Reply by goear** [reply]
Puedes poner un ejemplo de como queda tu composer.json?

**Reply by zaqueoae** [reply]
Disculpa, ya he entendido como va la cosa.  Te agradezco la respuesta.

**Comment by arraintxo** [comment]
Se han añadido ejemplos de uso en el README por lo que cierro esta discusión.
